platform/darwin/docs/theme/assets/css/jazzy.css.scss

```diff
@@ -367,6 +367,17 @@ pre code {
   color: $navigation_task_color;
 }
 
+.nav-group-task[data-url^="Protocols/"],
+.nav-group-task[data-name="MGLForegroundStyleLayer"],
+.nav-group-task[data-name="MGLMultiPoint"],
+.nav-group-task[data-name="MGLShape"],
+.nav-group-task[data-name="MGLSource"],
+.nav-group-task[data-name="MGLStyleLayer"],
+.nav-group-task[data-name="MGLTileSource"],
+.nav-group-task[data-name="MGLVectorStyleLayer"] {
```

Comment:
Embedding bits of our SDK into the jazzy theme’s CSS feels mildly icky — could this be scripted or upstreamed to jazzy?